src/main/webapp/query-control.js

```diff
@@ -193,3 +201,32 @@ function createStateDropdownList() {
     datalist.appendChild(optionElem);
   });
 }
+
+// On page load, show appropriate icon based on the localStorage
+function loadAppropriateIcon() {
+  const locationSettings = localStorage.getItem('locationSettings');
+  if (typeof locationSettings === 'undefined' || locationSettings === 'off') {
```

Comment:
If it's undefined, that means it hasn't been stored, right? In that case, would location settings not be on by default? I'm slightly confused by this because when I opened the app for the first time, I saw the icon without a slash, so I thought that meant it was on.

Reply:
It will be undefined when the user first opens the page and hasn't set the location on or off yet, so we leave it off to protect their privacy. That being said, it should not have been on when you opened the app for the first time. When I cleared localStorage and ran it, the location was off. Will look into why it was on for you